artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ClientSessionFactoryImpl.java

```diff
@@ -338,7 +338,7 @@ public ClientSession createSession(final boolean xa,
       return createSessionInternal(null, null, xa, autoCommitSends, autoCommitAcks, preAcknowledge, serverLocator.getAckBatchSize());
    }
 
-   // ConnectionLifeCycleListener implementation --------------------------------------------------
```

Comment:
where is the deprecation? I see this a fix, but I'm not sure why you're calling this a deprecation?